java/src/main/java/com/powsybl/python/dynamic/DynamicModelMapper.java

```diff
@@ -0,0 +1,71 @@
+package com.powsybl.python.dynamic;
```

Comment:
Don't forget to add license headers at some point ;)

Reply:
and SPDX in the copyright.
 /**
 * Copyright (c) 2022, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 *  SPDX-License-Identifier: MPL-2.0
 */